githublaronran.md

```diff
@@ -0,0 +1,4 @@
+# GitHub laronran
+
+* Name: LR
+* Location : Mountian view, CA
```

Comment:
The V in view should be capitalized. Other than that, great usage of capital letters.